cdk/typescript/README.md

```diff
@@ -0,0 +1,83 @@
+# Deploy The Lambda Power Tuner with CDK
```

Comment:
I'm wondering if this file should be moved up one folder? (for all the CDK projects across different languages)

See the C# PR here too: https://github.com/alexcasalboni/aws-lambda-power-tuning/pull/234